src/unix/linux_like/linux/mod.rs

```diff
@@ -3431,6 +3490,14 @@ extern "C" {
 
     pub fn shm_open(name: *const c_char, oflag: ::c_int, mode: mode_t) -> ::c_int;
 
+    pub fn statx(
+        dirfd: ::c_int,
+        pathname: *const c_char,
+        flags: ::c_int,
+        mask: ::c_uint,
+        statxbuf: *mut statx,
+    ) -> ::c_int;
+
```

Comment:
AFIAK the current version of musl (1.2.3) doesn't implement a wrapper for`SYS_statx`.
But, I think it's worth leaving the rest (struct statx, constant, etc) available for all linux target

Reply:
My mistake, you're correct.